linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_ca/bash/shared.sh

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_sle,multi_platform_ubuntu
+
+{{% if 'ubuntu' in product %}}
+{{{ bash_package_install("libpam-pkcs11") }}}
+{{% else %}}
+{{{ bash_package_install("pam_pkcs11") }}}
+{{% endif %}}
+
```

Comment:
you should consider the case when pam_pkcs11.conf does not exist, see previous ubuntu remediation